crypto/pkcs8/pkcs12_test.cc

```diff
@@ -140,6 +151,28 @@ TEST(PKCS12Test, TestNoEncryption) {
   TestImpl("kNoEncryption", StringToBytes(data), kPassword, nullptr);
 }
 
+// The AuthSafe field of the PFX is of type
+// ContentInfo https://datatracker.ietf.org/doc/html/rfc7292#appendix-D. It's
+// Content field is optional per
+// https://datatracker.ietf.org/doc/html/rfc2315#section-7, but we do not
+// support that. It must not be absent. Additionally, the Content field of
+// AuthSafe contains the AuthenticatedSafe
+// https://datatracker.ietf.org/doc/html/rfc7292#section-4.1; a sequence of
+// ContentInfo's, where each Content field is Optional, again per RFC2315. We do
+// not support this case either, the field cannot be absent.
+// Below two test fixtures validates the above. See V1217527752.
```

Comment:
Would referencing CVE-2024-0727 from OpenSSL provide more context for external consumers?